src/renderers/TextRenderer.js

```diff
@@ -31,10 +31,16 @@ class TextRenderer extends SketchRenderer {
       props.resizingConstraint
     );
 
-    const resolvedStyle = TextStyles.resolve(textStyle);
-    if (resolvedStyle) {
-      layer.style = resolvedStyle.sketchStyle;
-      layer.style.sharedObjectID = resolvedStyle.sharedObjectID;
+    const resolvedTextStyle = TextStyles.resolve(textStyle);
+    if (resolvedTextStyle) {
+      layer.style = resolvedTextStyle.sketchStyle;
+      layer.style.sharedObjectID = resolvedTextStyle.sharedObjectID;
+    } else {
+      const resolvedStyle = TextStyles.resolve(props.style);
+      if (resolvedStyle) {
+        layer.style = resolvedStyle.sketchStyle;
+        layer.style.sharedObjectID = resolvedStyle.sharedObjectID;
+      }
```

Comment:
Additionally, this could be flaky. It won't work if any additional style properties are set on the `Text` component. For example:

```jsx
    <Text style={{ alignSelf: 'flex-start', ...TextStyles.get('Headline') }}>
      Hello
    </Text>
```

Reply:
I think the real solution here would be to address why `flexDirection` is being set on the `textStyle`: https://github.com/airbnb/react-sketchapp/blob/1e3b24a2b293828027f35e3c491de8a04021e27e/src/buildTree.js#L35